src/unit_tests/wrappers/externals/cJSON/cJSON_wrappers.h

```diff
@@ -10,7 +10,7 @@
 #ifndef CJSON_WRAPPERS_H
 #define CJSON_WRAPPERS_H
 
-#include "external/cJSON/cJSON.h"
+#include <external/cJSON/cJSON.h>
```

Comment:
Why are we changing this? Isn't quotes specified for non-standard libraries in a specific location? I think it was correct before, letting the brackets for standard libraries.

Reply:
You are right. It was changed during a rebase. I noticed the change but did not bear in mind it wasn't the standard.
Fixed!